docs/excel/show-hide-add-in.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Show or hide an Office Add-in in a shared runtime
+description: 'Learn how to programmatically hide or show the UI of an add-in while it runs continuously'
+ms.date: 03/02/2020
+localization_priority: Normal
+---
+
+# Show or hide an Office Add-in in a shared runtime (preview)
+
+Office Add-ins can include any of the following parts:
+
+- a task pane
+- a UI-less function file
+- an Excel custom function
```

Comment:
I think each item in a list needs to be capitalized. The style guide doesn't explicitly state that, but every example is that way.